zboss/CHANGELOG.rst

```diff
@@ -9,6 +9,21 @@ Changelog
 
 All notable changes to this project in the |NCS| are documented in this file.
 
+Main branch
+***********
+
+All the notable changes included in the |NCS| main branch are documented in this section.
+
+Added
+=====
+
+* Precompiled libraries for experimental Zigbee Green Power Combo Basic functionality.
```

Comment:
If we removed the libs for ED, let's state here "for all device roles"?

Reply:
We did not remove any functionality by removing these libs, Proxy Basic functionality is not meant for End Devices anyway. Combo Basic functionality also is not meant for End Devices but we can specify Router and Coordinator roles